squire_lib/src/tournament.rs

```diff
@@ -918,6 +918,59 @@ impl Tournament {
         }
         digest
     }
+
+    /// Generates a round slip in HTML
+    pub fn round_slips_html(self, css: String) -> String {
+        let mut ret: String = "<!DOCTPYE HTML>".to_string();
+
+        ret += "<html lang=\"en\">";
+        ret += "<head><style>";
+        ret += &css;
+        ret += "</style></head>";
+        ret += "<body>";
+
+        for r in self.round_reg.rounds.values() {
```

Comment:
I don't think we want to iterate over all the round, only the active ones.
```suggestion
        for r in self.round_reg.rounds.values().filter(|r| r.is_active()) {
```